UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs

```diff
@@ -39,34 +38,14 @@ public struct TypeAndStrength
 		[Tooltip("How much damage or heals If negative per 1u")]
 		public float EffectPerOne;
 	}
-	
-	#region Emotes
 
-	public List<EmoteTypeAndChance> EmoteEffects = new List<EmoteTypeAndChance>();
-
-	[System.Serializable]
-	public struct EmoteTypeAndChance
-	{
-		public bool CustomEmote;
-		[Tooltip("the message only the person doing the emote can see")]
-		[ShowIf(nameof(CustomEmote))] [AllowNesting] public string CustomEmoterMessage;
-		[Tooltip("the emote those who are watching can see")]
-		[ShowIf(nameof(CustomEmote))] [AllowNesting] public string CustomShownMessage;
-		[HideIf("CustomEmote")] [AllowNesting] public EmoteSO Emote;
-		[Tooltip("Chance this action will happen every tick, first in the list rolls first")]
-		[Range(0,100)] [AllowNesting] public int ChancePerTick;
-		public bool StopIfOverdosed;
-	}
-	#endregion
-
-	[ShowNonSerializedField]
+	[System.NonSerialized]
 	public List<BodyPart> DamagedList = new List<BodyPart>(); //Not multithread safe
 
 	public override void PossibleReaction(List<BodyPart> senders, ReagentMix reagentMix,
-		float reactionMultiple, float BodyReactionAmount, float TotalChemicalsProcessed) //limitedReactionAmountPercentage = 0 to 1
+		float reactionMultiple, float BodyReactionAmount, float TotalChemicalsProcessed, out bool Overdose) //limitedReactionAmountPercentage = 0 to 1
```

Comment:
since you are here, can you make all the parameter names ``camelCase`` ?

Reply:
Can do